config/routes.js

```diff
@@ -46,62 +46,73 @@ module.exports.routes = {
    ***************************************************************************/
 
   // '/'                             :   'userController.homepage',
-  '/user/signup': 'userController.signup',
-  '/user/login': 'userController.login',
-  '/user/reLogin': 'userController.reLogin',
-  '/user/logout': 'userController.logout',
-  '/user/status': 'userController.status',
+  '/api/user/signup': 'userController.signup',
```

Comment:
When I moved to web history from hash history, it created a conflict in the route namespace. We use `/#/game/*` in vue but already used `/game/*` on sails
I had to move all of sails to a /api/* namespace